src/Entity/BillShipper.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/*
+ * Copyright © Eduard Sukharev
+ *
+ * For a full license, see the LICENSE file.
+ */
+
+namespace Ups\Entity;
+
+/**
+ * @author Eduard Sukharev <eduard.sukharev@opensoftdev.ru>
+ */
+class BillShipper
+{
+    /**
+     * @var string
+     */
+    private $accountNumber;
+
+    /**
+     * @var CreditCard
+     */
+    private $creditCard;
+
+    public function __construct($attributes = null)
+    {
+        if (isset($attributes->AccountNumber)) {
+            $this->setAccountNumber($attributes->AccountNumber);
+        }
+        if (isset($attributes->CreditCard)) {
+            $this->setAccountNumber(new CreditCard($attributes->CreditCard));
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function getAccountNumber()
+    {
+        return $this->accountNumber;
+    }
+
+    /**
+     * @param string $accountNumber
+     *
+     * @return BillShipper
+     */
+    public function setAccountNumber($accountNumber)
+    {
+        $this->accountNumber = $accountNumber;
+
+        return $this;
+    }
+
+    /**
+     * @return CreditCard
+     */
+    public function getCreditCard()
+    {
+        return $this->creditCard;
+    }
+
+    /**
+     * @param CreditCard $creditCard
+     * @return BillShipper
+     */
+    public function setCreditCard($creditCard)
```

Comment:
as per docs for **/ShipmentConfirmRequest/Shipment/PaymentInformation/Prepaid/BillShipper**:
`Must be the same UPS account number as the one provided in Shipper/ShipperNumber. Either this element or the sibling elements CreditCard or AlternatePaymentMethod must be provided, but all may not be provided`
and **/ShipmentConfirmRequest/Shipment/PaymentInformation/Prepaid/BillShipper/CreditCard**
`Required if neither of the following is present: - /ShipmentConfirmRequest/ Shipment/PaymentInformati on/Prepaid/BillShipper/Acco untNumber or /ShipmentConfirmRequest/ Shipment/PaymentInformati on/Prepaid/BillShipper/Alter natePaymentMethod. Credit card payment is valid for shipments without return service only`